docker/rholang-cli/rhoscala

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+input=$(realpath $1)
+
+docker run -ti -v ${input}:/tmp/input.rho rholang-cli
```

Comment:
This will need to change if we distribute the shell script and the container has a repo name on it (like `rchain/rholang-cli`).